operator/tests/central/basic/80-assert.yaml

```diff
@@ -1,10 +1,16 @@
-apiVersion: v1
```

Comment:
Any reason for these changes / renames? Why not use the `90` prefix for the telemetry tests?

Reply:
The previous `80` now `90` prefix is a deletion step and seemed to me like it was last intentionally.